### PR TITLE
update tpp image build

### PR DESCRIPTION
**Pull request by bloodearnest**
- tpp image: disable ubuntu user rather than remove.
- tpp image: update various docstrings.

### CODE REVIEW

File: tpp-backend/build-image.sh [hunk 1]
```diff
@@ -85,11 +76,12 @@ remove_cmd="Remove-VMHardDiskDrive -VMName $name -ControllerType IDE -Controller
...
 echo "Opening admin powershell..."
 
-# powershell -c start -verb runas is the magic to invoke the UAC dialog
 # Ideally, we'd just run these commands directly as an admin user, but I could
 # not figure out the quoting to pass the commands through.
+# powershell -c start -verb runas is the magic to invoke the UAC dialog
 powershell -Command start -verb runas powershell
 echo
-echo "To export $name VM image to $path, run following in Admin powershell"
+echo "To export $name VM image to $path, run following in Admin powershell (alredy copied to clipboard)"
```
Comment (madwort) [inline tpp-backend/build-image.sh:84]: typo `alredy`

File: tpp-backend/build-image.sh [hunk 2]
```diff
@@ -41,40 +42,30 @@ multipass exec $name -- sudo apt-get install -y linux-virtual-hwe-20.04 linux-to
...
-# not delete it via multipass exec.  Instead we ssh in with a different known
-# good user
-ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$sshuser@$vmaddress" -- sudo pkill -u ubuntu
-ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$sshuser@$vmaddress" -- sudo userdel --remove ubuntu
-
-# remove ssh host keys, cloud init will regenerate them on next boot
-# Note: this must be the the last thing we do!
-ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no "$sshuser@$vmaddress" -- sudo rm //etc/ssh/ssh_host_*_key*
+# disable the default ubuntu user
+# unfortunatlely, its very awkward for us to delete it here, due to requiring it to ssh in.
```
Comment (madwort) [inline tpp-backend/build-image.sh:66]: typo `unfortunatlely`

File: tpp-backend/remove-ubuntu-user.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+usermod -L ubuntu
+
+# expire the ubuntu user
+chage -E0 ubuntu
+
+# remove any ssh config
+rm -rf ~ubuntu/.ssh
+
+# remove the host ssh keys, we will no longer be able to ssh in after this
+rm /etc/ssh/ssh_host_*_key*
```
Comment (madwort) [inline tpp-backend/remove-ubuntu-user.sh:14]: verifying assumptions - do you need to restart ssh daemon after this? does it get auto-restarted?

**Review by madwort — CHANGES_REQUESTED** [review]

File: tpp-backend/remove-ubuntu-user.sh (hunk 3 above)
Comment (bloodearnest) [reply tpp-backend/remove-ubuntu-user.sh:14]: It doesn't get restarted, so the comment is kinda misleading, as you can ssh in if it hasn't been restarted.

But the next thing after this is to stop the VM, so it's not a problem.

I think probably best to just update the comment.

**Review by madwort — APPROVED** [review]

**Pull request merged**